cli.js

```diff
@@ -22,5 +22,14 @@ if (command === "list") {
 
 function runListCommand() {
   // Replace this with your implementation
-  console.log("Called the list command");
+  members.forEach(({firstName, lastName, jobTitle, location, userSlack}) =>
```

Comment:
Nice way to extract object keys

Reply:
@craig-dsilva Thanks for taking the time to do this for me.